packages/strapi-admin/ee/admin/containers/Roles/CreatePage/PluginsAndSettings/SubCategory/utils/formatActions.js

```diff
@@ -0,0 +1,30 @@
+import { get } from 'lodash';
+import { createArrayOfValues } from '../../../utils';
+
+/**
+ * We need to format the actions to an array of object that matches the ConditionsModal action's props
+ * @param {array<object>} actions
```

Comment:
I love this comments, thank you 🙏🏻 .

On a nitpicky note, what could be even more amazing would be to define the shape of the "object" in input and the object in the array in output.

Thus, just by reading this, we know what it is 😊